substrate/frame/message-queue/src/mock.rs

```diff
@@ -350,3 +350,7 @@ pub fn num_overweight_enqueued_events() -> u32 {
 		})
 		.count() as u32
 }
+
+pub fn fp(pages: u32, count: u64, size: u64) -> QueueFootprint {
+	QueueFootprint { fp: Footprint { count, size }, pages }
+}
```

Comment:
shouldn't this logic instead be expressed as a constructor for `QueueFootprint`?

also I think there should be a distinction between a `QueueFootprint` and `Footprint` in naming. 

e.g. instead of calling this `fp`, call it `queue_fp`

Reply:
Yea its kind of hacky, just wanted to get it compile 😆   
I can change the naming a bit.